### DISCUSSION TITLE
How to Include More Languages in My GitHub Repo's Languages Section?

### DISCUSSION THREAD

**Discussion by hafsaz17** [community/community#112502] [Other Feature Feedback, Questions, & Ideas] [answered]
![Screenshot_4](https://github.com/community/community/assets/159186250/0bfbce80-7917-4fdf-8194-3ff30b8dd049)
I currently have a repository on GitHub where only one language is showing in the 'Languages' section, even though I've used R and SQL in my code.  Is there a way to include these additional languages so the percentages accurately reflect the languages used in the project?

**Comment by ngljcb** [comment, marked as answer]
make sure your files have the right extension which is how github detects the languages, or in your case, you can try to manually specify some languages in `.gitattributes` file

**Reply by spenserblack** [reply]
Yup! Specifically, this should do it:
```gitattributes
# gitattributes
*.sql linguist-detectable
```

**Reply by ngljcb** [reply]
exactly!
